aedes.js

```diff
@@ -52,7 +52,7 @@ function Aedes (opts) {
     concurrency: opts.concurrency,
     matchEmptyLevels: true // [MQTT-4.7.1-3]
   })
-  this.handle = function handle (conn, req) {
+  this.handle = function handle (conn, req = null) {
```

Comment:
```suggestion
  this.handle = function handle (conn, req) {
```

Reply:
as mentioned in the docs, this is a fix for the optional parameter which is only used for websockets.

Reply:
@haci-xplora I know that but there is still no reason to set it to null as it will be undefined when omitted

Reply:
without using null (a default value), by looking at the code, a dev can falsely assume that it is a required parameter.

Reply:
req is not used in code anywhere, it is just set so outside aedes users can access it, when the type is ok in typescript vscode will tell them it is optional and that may not exists so the problem doesn't exist